pset-iterators.js

```diff
@@ -93,6 +107,14 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentialParties = presidents.filter((president) => {
+	return president.party
+}).map((president) => {
+    return president.party
+})
+
+console.log(presidentialParties)
+
```

Comment:
@cswebdesign Good work on this question here filtering out the president's party then chaining a map method to return only the president's parties in an array.

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)